Sources/ReadingListKit/ReadingListView.swift

```diff
@@ -5,12 +5,17 @@
 import SwiftUI
 
 public struct ReadingListView: View {
```

Comment:
Same consideration about view reusing here

Reply:
Similar to above, we can reuse suitable components as the view is built.